jumpscale/core/base/store/whooshfts.py

```diff
@@ -0,0 +1,139 @@
+from whoosh import fields
```

Comment:
what's the idea behind the filename? I don't undersdtand what fts means

Reply:
"fts" stands for full-text search :laughing: 

I used this not to conflict with the original `whoosh` module.